src/utils/latex.ts

```diff
@@ -0,0 +1,30 @@
+import { spawn } from 'node:child_process'
+import { mkdtemp } from 'node:fs/promises'
+import { createReadStream } from 'node:fs'
+import os from 'node:os'
+import path from 'node:path'
```

Comment:
Haven't seen this `node:*` style import before, do you have a reference for it?

Reply:
This is something I learned from https://twitter.com/rauschma/status/1516456272215826432

The [docs](https://nodejs.org/api/modules.html) say that support for the core module prefix was added in `v16.0.0` and `v14.18.0`